src/plot_api/plot_api.js

```diff
@@ -750,6 +751,17 @@ function cleanData(data, existingData) {
             }
         }
 
+        // fix typo in colorscale definition
```

Comment:
Agreed, just seems hacky to have it being handled in `plot_api.js` instead of in the `colorscale` component - if something uses colorscale but doesn't flow through this codepath, it won't get the same treatment, whereas I'd assume that there's a common entry for colorscale related things in `colorscale/____.js`.